feature/gnoi/packet_link_qualification/tests/packet_link_qualification_test/packet_link_qualification_test.go

```diff
@@ -265,7 +282,7 @@ func TestLinkQualification(t *testing.T) {
 				EndpointType: &plqpb.QualificationConfiguration_PacketGenerator{
 					PacketGenerator: &plqpb.PacketGeneratorConfiguration{
 						PacketRate: uint64(138888),
```

Comment:
```suggestion
						PacketRate: uint64(maxPPS),
```

Reply:
138888 PPS is used in base test https://github.com/openconfig/featureprofiles/blob/main/feature/gnoi/packet_link_qualification/tests/packet_link_qualification_test/packet_link_qualification_test.go#L267 right now.

Also not sure using MaxPPS will work with 100G link, even with below packet parameters it will be more than the bandwidth of a 100G link ?

40000000(PPS) *512 (Bytes) = 20480000000 Bps => 163.84 Gbps